resources/views/partials/example-requests/python.blade.php

```diff
@@ -2,17 +2,17 @@
 import requests
 import json
 
-url = '{{ trim(config('app.docs_url') ?: config('app.url'), '/')}}/{{ ltrim($route['uri'], '/') }}'
-@if(count($route['bodyParameters']))
+url = '{{ rtrim($baseUrl, '/') }}/{{ ltrim($route['boundUri'], '/') }}'
+@if(count($route['cleanBodyParameters']))
 payload = {
-	@foreach($route['bodyParameters'] as $attribute => $parameter)
+	@foreach($route['cleancleanBodyParameters'] as $attribute => $parameter)
```

Comment:
A Find & Replace gone wrong, I see. 😆

```suggestion
	@foreach($route['cleanBodyParameters'] as $attribute => $parameter)
```

Reply:
Sorry... Exactly this!